lib/ember_cli/manifest.rb

```diff
@@ -0,0 +1,15 @@
+require "sprockets"
+
+module EmberCli
+  class Manifest
+    def initialize(environment, path)
+      require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.